src/js/components/locations-configuration/LoadDataPage.jsx

```diff
@@ -90,31 +97,72 @@ const LoadDataContent = ({
       {ReactHtmlParser(summaryItemsList)}
     </div>
     <div className="d-flex justify-content-between m-3">
-      <button type="button" onClick={stepBackHandler} className="btn btn-outline-primary">
+      <button type="button" onClick={() => nextStep(null)} className="btn btn-outline-primary">
```

Comment:
Add a `prevStep` handler. (Bring back `stepBackHandler` but as a `prevStep`). Unless you want to unify it under one handler, then call it `goToStep`.

Reply:
yes, I wanted to unify it as one handler and I agree **goToStep** make a lot more sens in that case than **nextStep**